clang/lib/Headers/hlsl/hlsl_detail.h

```diff
@@ -13,23 +13,23 @@ namespace hlsl {
 
 namespace __detail {
 
-#define _HLSL_INLINE                                                           \
-  __attribute__((__always_inline__, __nodebug__)) static inline
```

Comment:
Note that while constexpr implies `inline` and since it also implies `const` dropping `static here is fine too, we are dropping the two attributes here. `__always_inline__` shouldn't matter, but dropping `nodebug` presumably does have an effect. I don't fully understand what `nodebug` is gaining us though, and if we really want to use it we should probably do that more systematically in this header anyway, so this all seems fine.